cohortApp/db/migrations/20191217221201_createCohorts.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable("cohorts", table => {
+      table.increments("id"); 
+      table.string("name")
+      table.text("members");
+      table.string("logoUrl")
```

Comment:
Use `table.text` instead of string for the logoUrl to use a data type that accepts a length that's greater than 255, since image urls can get long.